lesson02/elb_variables.tf

```diff
@@ -19,24 +19,21 @@ variable "elb_name" {
 }
 
 variable "elb_listener" {
-  default = [
-    {
-      instance_port     = "80"
+  default = {
+      instance_port     = 80
       instance_protocol = "HTTP"
-      lb_port           = "80"
+      lb_port           = 80
       lb_protocol       = "HTTP"
-    },
-  ]
+  }
 }
 
 variable "elb_health_check" {
-  default = [
-    {
+  default = {
       target              = "HTTP:80/"
       interval            = 30
       healthy_threshold   = 2
       unhealthy_threshold = 2
       timeout             = 5
-    },
-  ]
+  }
 }
+
```

Comment:
styling: let's remove the extra line

Reply:
we can do this in the next cycle this is not a blocker, thanks for pointing that out

Reply:
This will be done in another PR. Created an issue to keep track of this #33